main.go

```diff
@@ -206,14 +209,29 @@ func getLatestTag(c config) (string, error) {
 
 	}
 
+	baseVersion, _ := getVersion(c)
 	// turn the array into a new collection of versions that we can sort
 	var versions []*version.Version
 	for _, raw := range versionsRaw {
-		v, _ := version.NewVersion(raw)
-		if v != nil {
-			versions = append(versions, v)
+		//if same-release argument is set work only with versions which Major and Minor versions are the same
+		if c.samerelease == true {
```

Comment:
I'm no go expert so I don't know the best idiomatic way of doing it. But wouldn't be simpler just to
```
if c.samerelease {
```